### ISSUE TITLE
Challenge info always shows 'SuperUser' as contact

### ISSUE THREAD

**Issue by mvexel** [maproulette/maproulette2#234]
See here:

![](https://www.dropbox.com/s/rtcih6ffa89hmmj/Screenshot%202016-12-09%2012.54.27.png?raw=1)

**Comment by mgcuthbert** [comment]
Yes, we the new feature for the owner associated with the challenge was introduced the SuperUser essentially became the owner of all challenges. This is because we didn't know the owner previously because the feature didn't exists, so it just defaulted to SuperUser. 

The owner can be manually deduced, or we could write a SQL query that would figure out the owner based on the home folder. Any newly created challenges will contain the owner that created them though.
